snorkel/analysis/metrics.py

```diff
@@ -18,7 +18,7 @@ def metric_score(
     preds: np.ndarray,
     probs: np.ndarray,
     metric: str,
-    filter_dict: Optional[Dict[str, List[int]]] = None,
+    filter_dict: Dict[str, List[int]] = {"golds": [0]},
```

Comment:
No mutables for defaults plz (https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Note to self: this isn't caught by flake8, but we can with an add-on https://github.com/pycqa/flake8-bugbear